debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -851,7 +851,7 @@ private static int validateMaxQueueSize(Configuration config, Field field, Field
         return count;
     }
 
-    protected static int validateSkippedOperation(Configuration config, Field field, ValidationOutput problems) {
+    public static int validateSkippedOperation(Configuration config, Field field, ValidationOutput problems) {
```

Comment:
@rajdangwal  do you think we should add a comment here to revert the change when we bump the jdk version in future?